src/main/java/ro/societateahermes/backendservice/service/CDMemberServiceInterface.java

```diff
@@ -0,0 +1,16 @@
+package ro.societateahermes.backendservice.service;
+
+import ro.societateahermes.backendservice.entities.CDMember;
+import ro.societateahermes.backendservice.entities.DTO.CDMemberDTO;
+
+import java.util.List;
+
+public interface CDMemberServiceInterface {
+    CDMember save(CDMember CDMember);
```

Comment:
dto

Reply:
@McShenanigans  In service ar trebui sa ajunga obiectul propriu zis, acele DTO-uri ar trebui sa fie la nivel de controller

Reply:
pe service am lasat doar obiecte, iar pe controller am dto s